stage1/rootfs/usr_from_src/Makefile

```diff
@@ -84,10 +85,10 @@ systemd.src: Makefile patches/*
 	{ [ ! -e systemd ] || rm -Rf systemd; }
 	mkdir systemd
 	if [ "$(RKT_STAGE1_SYSTEMD_VER)" = "HEAD" ]; then \
-		git clone --depth 1 $(RKT_STAGE1_SYSTEMD_SRC) ; \
```

Comment:
I wrote it in the commit message:
> Unfortunately, git-describe does not always work well on repository
> cloned with --depth so I am removing the --depth optimization.

When cloning the "master" branch (or just HEAD), it might not have a tag, so git-describe is unable to describe it. `git clone --no-single-branch` does not help because even though it fetches all the tags, the commits between tags are squashed so git-describe does not give the correct answer.